backend/app/models/user.ts

```diff
@@ -12,10 +12,20 @@ const AuthFinder = withAuthFinder(() => hash.use('scrypt'), {
 
 export default class User extends compose(BaseModel, AuthFinder) {
   @column({ isPrimary: true })
-  declare id: string
+  declare id: number
 
   @column()
-  declare password: string
+  declare usosId: string
+  @column()
+  declare firstName: string
+  @column()
+  declare lastName: string
+  @column()
+  declare studentNumber: string
+  @column()
```

Comment:
why columny do accesstokenu i sekretu
1. tego nie storujesz nawet, w routcie loginu
2. takich rzeczy nie chcemy miec w bazie, to jest za wrazliwe, tak jak hasla hashujesz to tego nie trzymasz, bo to nie jest nam potrzebne

Reply:
okeeej, git cos nie zalapal z zaznaczeniem, chodzilo mi ofc o pola tokenow